[2024-01-30T17:38:55.554+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-30T08:00:00+00:00 [queued]>
[2024-01-30T17:38:55.588+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-30T08:00:00+00:00 [queued]>
[2024-01-30T17:38:55.590+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-30T17:38:55.858+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-01-30 08:00:00+00:00
[2024-01-30T17:38:55.871+0000] {standard_task_runner.py:60} INFO - Started process 3001 to run task
[2024-01-30T17:38:55.899+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-01-30T08:00:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/weather_api_dag.py', '--cfg-path', '/tmp/tmppunv7uzr']
[2024-01-30T17:38:55.904+0000] {standard_task_runner.py:88} INFO - Job 87: Subtask get_weather_data
[2024-01-30T17:38:56.066+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-30T08:00:00+00:00 [running]> on host 68c8327b56d3
[2024-01-30T17:38:56.390+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-30T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-30T08:00:00+00:00'
[2024-01-30T17:38:56.940+0000] {logging_mixin.py:188} INFO - {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}
[2024-01-30T17:39:01.039+0000] {logging_mixin.py:188} INFO - {'record': {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}, 'metadata': {'id': '65b93435dc746540189de44d', 'createdAt': '2024-01-30T17:39:01.062Z', 'private': True, 'name': 'weather_959b7e1b-ae96-44be-b3ac-f01aed9cff01', 'collectionId': '65ab4466266cfc3fde7c6f31'}}
[2024-01-30T17:39:01.046+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-30T17:39:01.074+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240130T080000, start_date=20240130T173855, end_date=20240130T173901
[2024-01-30T17:39:01.133+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-30T17:39:01.187+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
